src/travertino/node.py

```diff
@@ -118,3 +123,10 @@ def refresh(self, viewport):
             self.style.layout(self, viewport)
             if self.applicator:
                 self.applicator.set_bounds()
+
+    @classmethod
```

Comment:
Why a class method, rather than an instance method? The fact this used to be a standalone function means a class method is the most *literal* interpretation, but it's very much an operation on an instance, so we might as well preserve that relationship.